sdk/appconfiguration/app-configuration/src/internal/helpers.ts

```diff
@@ -0,0 +1,69 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { ETagOption, ListConfigurationSettingsOptions, AppConfigurationGetKeyValuesOptionalParams } from '..';
+import { URLBuilder } from '@azure/core-http';
+import { isArray } from 'util';
+
+/**
+ * Formats the etag so it can be used with a if-match header
+ * @internal
+ * @ignore
+ */
+export function formatETagForMatchHeaders(objectWithEtag: ETagOption): string | undefined {
+  if (objectWithEtag.etag) {
+    return `"${objectWithEtag.etag}"`;
+  }
+
+  return undefined;
+}
+
+/**
+ * Transforms the keys/labels parameters in the listConfigurationSettings and listRevisions
+ * into the format the REST call will need.
+ * 
+ * @internal
+ * @ignore
+ */
+export function formatWildcards(
+  listConfigOptions?: ListConfigurationSettingsOptions
+): Pick<AppConfigurationGetKeyValuesOptionalParams, "key" | "label"> {
+  if (listConfigOptions == null) {
+    return {};
+  }
+
+  let key = undefined;
```

Comment:
explicitly assigning undefined feels like overkill, since it's the default value of any param. Rather, I'd prefer to see a type annotation for what its valid values are, e.g. `let key: string | undefined;`

Reply:
I agree. Removed.